<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpServer</class>
 <widget class="QWidget" name="UdpServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="txtMain">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="ckhexsend">
        <property name="text">
         <string>十六进制发送</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckhexread">
        <property name="text">
         <string>十六进制接收</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labListenIP">
        <property name="text">
         <string>监听地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ipcbox"/>
      </item>
      <item>
       <widget class="QLabel" name="labListenPort">
        <property name="text">
         <string>监听端口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="port"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>目标地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="targetAddr"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>目标端口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="targetPort"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnListen">
        <property name="text">
         <string>监听</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemove">
        <property name="text">
         <string>移除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveconfigbtn">
        <property name="text">
         <string>保存参数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labCount">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>共 0 个客户端</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item>
       <widget class="QCheckBox" name="cknew">
        <property name="text">
         <string>新连接</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckSelectAll">
        <property name="text">
         <string>对所有客户端发送</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="textEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnSend">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
